ecs_fargate/manifest.json

```diff
@@ -2,7 +2,8 @@
   "categories": [
     "aws",
     "containers",
-    "orchestration"
+    "orchestration",
+    "log collection"
```

Comment:
> You can monitor Fargate logs by using the AWS FireLens integration built on Datadogs Fluentbit output plugin to send logs to Datadog, or by using the `awslogs` log driver and a Lambda function to route logs to Datadog. Datadog recommends using AWS FireLens because you can configure Fluent Bit directly in your Fargate tasks.

Same comment as cassandra_nodetool,  does it make sense to categorize it as `log collection`?